data/dataspace.json

```diff
@@ -11,7 +11,96 @@
     "members": 1178,
     "datasets": 822,
     "services": 674,
-    "governanceUrl": "https://xd.adobe.com/view/31a3d2a5-9f07-4e31-a612-20059ff929a5-64f0/screen/d8a45cb1-7433-40b3-b6ce-4237bfcd0678/?fullscreen",
+    "governance": {
+      "accessControl": [
+        {
+          "subCategory": "data spaces",
+          "rules": [
+            {
+              "title": "authentification",
```

Comment:
Typo 😉 
```suggestion
              "title": "authentication",
```

Reply:
Good catch. I've also changed Go to GB as the rules are in english 👍